CHANGELOG

```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* Removed assertions of cluster rebalance js test that obligated the rebalance
+  plan to have moves, but there were cases in which all the possible moves 
+  were negative, so there was no actual better move to perform.
```

Comment:
You are referring to the scores here? I think that is a different thing.
```suggestion
* Removed assertions from cluster rebalance js test that obligated the rebalance
  plan to always have moves, but there were cases in which all there are none.
```

Reply:
Yes, but it got confusing